geomesa-tools/bin/install-hadoop-accumulo.sh

```diff
@@ -21,20 +21,113 @@ guava_version="11.0.2"
 com_log_version="1.1.3"
 commons_vfs2_version="2.0"
 
+# Resource download location
+base_url="https://search.maven.org/remotecontent?filepath="
+
+# Print versions available for download
+function printVersions() {
+  # usage: getVersions [url]
+  content=$(wget $1 -q -O -)
+  versions=$(echo "${content}" | grep -oP ">[0-9]{1,}\.[0-9]{1,}\.[0-9]{1,}\/<" | grep -oP "[0-9]{1,}\.[0-9]{1,}\.[0-9]{1,}")
+  versionArray=($(echo "$versions" | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/ /g'))
+  size=${#versionArray[@]}
+  i=0
+  while [[ $i -lt $size ]]; do
+    echo -e "${versionArray[i]}\t${versionArray[i+1]}\t${versionArray[i+2]}\t${versionArray[i+3]}"
+    i=$(expr $i + 4)
+  done
+}
+
+# Command Line Help
+NL=$'\n'
+usage="usage: ./install-hadoop-accumulo.sh [[target dir] [<version(s)>]] | [-g|--get-versions] | [-h|--help]"
+
+# Parse command line options
+if [[ "$1" == "-h" || "$1" == "--help" ]]; then
+	echo "${usage}"
+	echo "${NL}"
+	echo "All versions are detected automatically at compile time."
+	echo "These parameters are for situations where this may need overwritten."
+	echo "${NL}"
+	echo "Options:"
+	echo "  -a,--accumulo-version     Manually set Accumulo version"
+	echo "  -h,--hadoop-version       Manually set Hadoop version"
+	echo "  -z,--zookeeper-version    Manually set Zookeeper version"
+	echo "  -t,--thrift-version       Manually set Thrift version"
+	echo "  -g,--get-versions         Print out available version numbers."
+	echo "${NL}"
+	echo "Example:"
+	echo "./install-hadoop-accumulo.sh /opt/jboss/standalone/deployments/geoserver.war/WEB-INF/lib -a 1.7.1 -h 2.7.3"
+	echo "${NL}"
+	exit
+elif [[ "$1" == "-g" || "$1" == "--get-versions" ]]; then
+  accumulo_version_url="${base_url}org/apache/accumulo/accumulo/"
+  hadoop_version_url="${base_url}org/apache/hadoop/hadoop-main/"
+  zookeeper_version_url="${base_url}org/apache/zookeeper/zookeeper/"
+  thrift_version_url="${base_url}org/apache/thrift/libthrift/"
+
+  echo "Available Accumulo Versions"
+  printVersions "${accumulo_version_url}"
+  echo "Available Hadoop Versions"
+  printVersions "${hadoop_version_url}"
+  echo "Available Zookeeper Versions"
+  printVersions "${zookeeper_version_url}"
+  echo "Available Thrift Versions"
+  printVersions "${thrift_version_url}"
+
+  exit
+else
+  install_dir=$1
+  shift
+fi
+
+while [[ $# -gt 1 ]]; do
+	key="$1"
+
+	case $key in
+		-a|--accumulo-version)
+			accumulo_version="$2"
+			shift
+		;;
+		-h|--hadoop-version)
```

Comment:
Is there a reason why "-h" is used here in addition to being used as an alternative for "--help"?